config/project-scratch-def.json

```diff
@@ -2,7 +2,7 @@
     "orgName": "Easy Spaces",
     "edition": "Developer",
     "hasSampleData": false,
-    "features": ["ServiceCloud"],
+    "features": ["Walkthroughs"],
```

Comment:
@adityanaag3 FYI - I found that we do not need the ServiceCloud enabled for this app at all. Fixing it to just use the Walkthroughs feature.